.github/workflows/continous-integration.yml

```diff
@@ -47,72 +53,28 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           filters: .github/change_filters.yml
 
-  documentation:
-    name: Test Documentation
+
+  wait_for_docs_tests:
+    # Looks for doc test workflows and waits for it to complete successfully
+    # Runs on tag pushes and pushes to main exclusively, as it is a dependency of release jobs
+    name: Wait for docs tests
     runs-on: ubuntu-latest
-    needs: [changes]
+    if: github.event_name == 'push' && (startsWith(github.event.ref, 'refs/tags') || github.ref == 'refs/heads/main')
```

Comment:
Why do we need it? I see that the `Test Documentation` runs always, thus we should have this `if` here

Reply:
You mean we don't need the if in line 62? I agree. Should I delete it?